### ISSUE TITLE
Sonar-CXX sensor seems not working since upgrade to SonarQube 7.9.2

### ISSUE THREAD

**Issue by vhuynh4thalesgroup** [SonarOpenCommunity/sonar-cxx#1938]
### Description

Since migration to SonarQube 7.9.2, we encounter a problem with SonarQube CXX plugin.
It does not raise any issue anymore and the source code is not retrieved.
We use it to import issues raised by Coverity.

Context : multi-languages project (java => SonarJava, ada => GnatCheck, c++ => Coverity + parsing through cxx plugin).

What we already did :
- upgrade sonar-cxx plugin to 1.3.1.
- configure "other rules" based on coverity rules definition.
- set up new c++ quality profiles (Coverity way) => our 639 rules based on coverity rules definition

We run SonarQube analysis from gradle.

### Steps to reproduce the problem

Unfortunately, I cannot provide the whole set of files.
But I can provide the execution logs and the SonarQube scanner context.

### Expected behavior

C++ source code parsed (currently no C++ source code detected).
Issues detected by the CXX plugin.

### Actual behavior

Empty source code and violations not imported while the sonar.sources targets an existing folder and sonar.cxx.other.reportPaths targets an existing file (by the way shall we use reportPath or reportPaths ?).

```
Scanner properties of module: com.thalesgroup.las:itfcodegen:unifiedtools:cpp
  - sonar.coverity.cov-cpp.suffixes=.disabled
  - sonar.coverity.enable=false
  - sonar.cxx.coverage.reportPaths=build/gcov/gcovreport.xml
  - **sonar.cxx.other.reportPaths=build/coverity/checkreport.xml**
  - sonar.cxx.suffixes.headers=.hxx,.hpp,.hh,.h
  - sonar.cxx.suffixes.sources=.cxx,.cpp,.cc,.c
  - sonar.cxx.xunit.reportPaths=build/cppunit/report/report.xml
  - sonar.cxx.xunit.xsltURL=cppunit-1.x-to-junit-1.0.xsl
  - sonar.moduleKey=com.thalesgroup.las:itfcodegen:unifiedtools:cpp
  - **sonar.projectBaseDir=/projets/stdcomp/members/t0174267/git/unifiedtools/cpp**
  - sonar.projectKey=com.thalesgroup.las:itfcodegen:unifiedtools:cpp
  - sonar.projectName=unified-tools-cpp
  - sonar.projectVersion=3.0.0-RC3-SNAPSHOT
  - **sonar.sources=/projets/stdcomp/members/t0174267/git/unifiedtools/cpp/src**
  - sonar.tests=tests
  - sonar.xml.file.suffixes=.disable
```

### Related information

* cxx plugin version : 1.3.1
* SonarQube version : 7.9.2

**Comment by vhuynh4thalesgroup** [comment]
Sorry, I have difficulties to attach files.
I will do it as soon as possble.

**Comment by vhuynh4thalesgroup** [comment]
Sorry, I have difficulties to attach files.
I will do it as soon as possible.

**Comment by vhuynh4thalesgroup** [comment]
Sorry, I have difficulties to attach files.
I will do it as soon as possible.

**Comment by vhuynh4thalesgroup** [comment]
Sorry, I have difficulties to attach files.
I will do it as soon as possible.

**Comment by vhuynh4thalesgroup** [comment]
Sorry, I have difficulties to attach files.
I will do it as soon as possible.

**Comment by guwirth** [comment]
Hi @vhuynh4thalesgroup,

> Since migration to SonarQube 7.9.2

From which version are you coming?

> by the way shall we use reportPath or reportPaths ?

Use https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Supported-configuration-properties#sonarcxxotherreportpath

Please look here first:
* to get more info  https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Get-debug-information
* the source file should be indexed
* should find the report
* rules must be defined and activated in your profile https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Extending-the-code-analysis#usage-of-unsupported-code-checkers
* path of issues must match https://github.com/SonarOpenCommunity/sonar-cxx/wiki/Path-and-path-separator-issues

Regards,
